src/android/patches/makeStringsPatch.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function makeStringsPatch(params) {
+  const patch = Object.keys(params).map(key =>
+    `    <string moduleConfig="true" name="${key}">${params[key]}</string>`
```

Comment:
I generally agree it might be hard to see the problem / solution at this stage, so let's just leave this for now as is and maybe come back in the future once we have more background!

You win 💃